apikey-service/src/main/java/eu/europeana/apikey/keycloak/KeycloakManager.java

```diff
@@ -543,15 +543,16 @@ public boolean isClientAuthorized(String apikey, KeycloakAuthenticationToken key
      *
      * @param enable enable or disable a client
      * @param clientId client identifier
+     * @param apikeyUpdate update information which will be used in case of enabling
      * @param securityContext security context with access token
      * @throws ApikeyException when client not found in Keycloak or update failed
      */
-    public void enableClient(boolean enable, String clientId, KeycloakSecurityContext securityContext) throws ApikeyException {
+    public void enableClient(boolean enable, String clientId, ApikeyDetails apikeyUpdate, KeycloakSecurityContext securityContext) throws ApikeyException {
```

Comment:
Why add the apikeyUpdate field if it's not used?

Reply:
Normally you would be right but this method was introduced to be used both for enable and disable clients. I simply introduced this parameter in order to avoid changing the method signature in the very next implementation task (EA-1614 reenable client). If that's necessary I can remove it here but after merging code from EA-1614 it will be back anyway...